src/twisted/topfiles/8877.bugfix

```diff
@@ -0,0 +1 @@
+Twisted now can resolve names with IPv6 DNS servers
```

Comment:
The style guide suggests the subject of the news fragment should be a FQPN.  So I'd say "twisted.names.client.Resolver can now ...".  Also, the fragment should be a complete sentence - so it should end with a `.`.